lib/tests/fetch.rs

```diff
@@ -21,13 +22,17 @@ async fn create_relate_select() -> Result<(), Error> {
 		SELECT *, ->bought->product.* AS products FROM user;
 		SELECT *, ->bought AS products FROM user FETCH products;
 		SELECT *, ->(bought AS purchases) FROM user FETCH purchases, purchases.out;
-		LET $param1 = \"purchases\";
+		LET $param1 = 'purchases';
 		SELECT *, ->(bought AS purchases) FROM user FETCH $param1, purchases.out;
+		LET $param2 = purchases;
```

Comment:
@tobiemh Are you happy with this?

```rust
LET $param2 = purchases;
```

Or do we want something more strict?

Reply:
This should result in `NONE` right? There's no document here from which `purchases` can be inherited